### ISSUE TITLE
Album applet

### ISSUE THREAD

**Issue by Ammako** [exelix11/SwitchThemeInjector#104]
Pretty niche use case, but for completeness' sake it would be nice if we could add background image to the album applet too (0x010000000000100D)

Most people never use the album, but for those who are interested in using it, it would be nice, if it could have a background image to match the rest of the system.

**Comment by DraconicNEO** [comment]
I agree I think that this is a very much needed feature, I think it's kind of strange that this was overlooked since there is theming support for the news applet but not the album applet.

I find it really strange that people don't use the album applet, do people just not take screenshots on their switch anymore? I know I do and I think that it's kind of sad that there is no custom theme support for the Album applet.

@exelix11 I really hope you eventually implement this feature in the future, I'm guessing it would probably be very easy since it likely would be able to use much of the same code as the theming for the news applet.
